commnad_line_rock_paper_scissors.py

```diff
@@ -15,19 +15,32 @@ def main():
         game.rounds = input('How many odd rounds would you like to play? ')
 
     while not game.is_complete:
-        choices = ', or '.join(
-            [
-                ', '.join(game.valid_choices[:-1]),
-                game.valid_choices[-1]
-            ]
-        )
-        human.choice = input('Choose {}: '.format(choices))
-        computer.choice = random.choice(game.valid_choices)
+        validate_users_choice(game)
+
         game.print_players_menu('Input')
         game.play()
         game.print_players_menu('Score')
 
     game.determine_winner()
 
+
+def validate_users_choice(game):
```

Comment:
This function name really only describes a subset of what's going on within the function, and not even the "ultimate" operation either (which in this case would be assigning moves to players).

This suggests to me that the function is doing too much. I see the following steps taking place within this function:
1. Prompt a human CLI player for input
2. Randomly select a choice for a computer player
3. Convert each of the chosen values to a `Move`
4. Assign the converted `Move` values to the appropriate player

I'll follow up on this more in my overall comment on the PR review.